components/motor/gpio/encoded.go

```diff
@@ -311,11 +314,10 @@ func (m *EncodedMotor) goForInternal(rpm, goalPos, direction float64) error {
 // towards the specified target/position
 // This will block until the position has been reached.
 func (m *EncodedMotor) GoTo(ctx context.Context, rpm, targetPosition float64, extra map[string]interface{}) error {
-	pos, err := m.position(ctx, extra)
+	currRotations, err := m.Position(ctx, extra)
 	if err != nil {
 		return err
 	}
-	currRotations := pos / m.ticksPerRotation
 	rotations := targetPosition - currRotations
```

Comment:
math.Abs(rpm) will likely fix goto.

Reply:
gonna be putting up a separate pr for this one as well as update some of the other drivers